deployment/CP05MOAS-GL380_Deploy.csv

```diff
@@ -65,9 +65,9 @@ TS20211214,,TS20220221,,CP05MOAS-GL380-02-FLORTM000,11,1,2021-12-15T07:36:00,202
 TS20211214,,TS20220221,,CP05MOAS-GL380-03-CTDGVM000,11,1,2021-12-15T07:36:00,2022-02-22T08:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-CTDGVM-00073,39.99217,-70.30902,,1000,1000,FZ-1,,
 TS20211214,,TS20220221,,CP05MOAS-GL380-04-DOSTAM000,11,1,2021-12-15T07:36:00,2022-02-22T08:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-DOSTAM-00496,39.99217,-70.30902,,1000,1000,FZ-1; top mount,,
 TS20211214,,TS20220221,,CP05MOAS-GL380-05-PARADM000,11,1,2021-12-15T07:36:00,2022-02-22T08:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50244,39.99217,-70.30902,,1000,1000,FZ-1,,
-AR66,,,,CP05MOAS-GL380-00-ENG000000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGCON-GLDRCP-00380,39.93425,-70.58905,,1000,1000,FZ-2,,
-AR66,,,,CP05MOAS-GL380-01-ADCPAM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-ADCPAM-50867,39.93425,-70.58905,,1000,1000,FZ-2,,
-AR66,,,,CP05MOAS-GL380-02-FLORTM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-FLORTM-04251,39.93425,-70.58905,,1000,1000,FZ-2,,
-AR66,,,,CP05MOAS-GL380-03-CTDGVM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-CTDGVM-09037,39.93425,-70.58905,,1000,1000,FZ-2,,
-AR66,,,,CP05MOAS-GL380-04-DOSTAM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-DOSTAM-00171,39.93425,-70.58905,,1000,1000,FZ-2,,
-AR66,,,,CP05MOAS-GL380-05-PARADM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50152,39.93425,-70.58905,,1000,1000,FZ-2,,
\ No newline at end of file
+AR66,,,,CP05MOAS-GL380-00-ENG000000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGCON-GLDRCP-00380,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL380-01-ADCPAM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-ADCPAM-50867,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL380-02-FLORTM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-FLORTM-04251,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL380-03-CTDGVM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-CTDGVM-09037,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL380-04-DOSTAM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-DOSTAM-00171,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL380-05-PARADM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50152,39.93425,-70.58905,,1000,1000,FZ-1,,
```

Comment:
good - moved to FZ-1 as noted in slack channel.

Reply:
Looks good.